lib/expressroutes.js

```diff
@@ -151,9 +175,18 @@ function expressroutes(router, options) {
         res.json(options.api);
     });
 
+    var routesMethod = {};
+
     routes.forEach(function(route) {
         makeExpressRoute(router, mountpath, route);
+        var path = buildRoutePath(mountpath, route.path);
+        routesMethod[path] = routesMethod[path] || [];
+        routesMethod[path].push(route.method.toLowerCase());
     });
+
+    for(var path in routesMethod) {
```

Comment:
Prefer `Object.keys...forEach` to `for...in`. Also white space after `for`, `if`, `function` (anonymous), etc...